mcs/class/corlib/System.Reflection.Emit/CustomAttributeBuilder.cs

```diff
@@ -57,7 +62,20 @@ public class CustomAttributeBuilder : _CustomAttributeBuilder {
 		
 		[MethodImplAttribute(MethodImplOptions.InternalCall)]
 		static extern byte[] GetBlob(Assembly asmb, ConstructorInfo con, object[] constructorArgs, PropertyInfo[] namedProperties, object[] propertyValues, FieldInfo[] namedFields, object[] fieldValues);
-		
+
+		internal object Invoke ()
```

Comment:
This looks wrong, a *Builder should not be creating things, that's for the concrete versions in SR.

Reply:
Could you clarify the changes you'd like me to make to this PR a bit?